pravega/pravega
# Issue 5931: Key-Value Tables Beta 2

**Change log description**  
- Key-Value Tables Beta 2. 
- See [PDP-48](https://github.com/pravega/pravega/wiki/PDP-48-Key-Value-Tables-(Beta-2)).
- Merging feature branch `feature-kvt-beta2` into master.

**Purpose of the change**  
Fixes #5931.

**What the code does**  
 See [PDP-48](https://github.com/pravega/pravega/wiki/PDP-48-Key-Value-Tables-(Beta-2)).
This branch collected several reviewed PRs. There is nothing new in this branch that hasn't yet been reviewed.

**How to verify it**  
Build must pass.
Verify commit history into this branch that it is only made of:
- Mergers from master (plus an additional conflict resolution commit immediately afterwards)
- Mergers of approved PRs into it.
